src/Core/functions.php

```diff
@@ -153,13 +153,9 @@ function pr($var)
      */
     function pj($var)
     {
-        if (!Configure::read('debug')) {
```

Comment:
This should not be removed. Throughout the core we try and return early from method when possible.